### ISSUE TITLE
Make readme.md less painful to read

### ISSUE THREAD

**Issue by UbnGithub** [AlphaDevelopmentTeam/AlphaBot#4]
Instead Of Typing Like This You Should Start Typing like this as it's a lot more natural to read and makes sense.

**Comment by EnterNewName** [comment]
yes i always never make sense while writing a document or etc, i will get this done asap
